lib/operations/mongo_client_ops.js

```diff
@@ -439,10 +439,12 @@ function createTopology(mongoClient, topologyType, options, callback) {
     }
 
     const MongoClient = loadClient();
-    const connectionString =
-      os.platform() === 'win32'
-        ? 'mongodb://localhost:27020/?serverSelectionTimeoutMS=1000'
-        : 'mongodb://%2Ftmp%2Fmongocryptd.sock/?serverSelectionTimeoutMS=1000';
+    let connectionString =
```

Comment:
I typically try to avoid nested ternary expressions. Can we rewrite this as if statements? Worst case, can we break it out into its own function?

Reply:
rewritten as if statements.